test/deploy/onboarding-tests/roles/configure/tasks/main.yml

```
@@ -0,0 +1,36 @@
+---
+- debug:
+    msg: Install Cassandra
+
+- name: Trigger Onboarding tests
+  shell: "git clone https://${ENV_SECRET_USERNAME}:${ENV_SECRET_TOKEN}@github.com/newrelic/onboarding-e2e-tests.git"
```

```suggestion
   - name: Clone a github repository
     git:
       repo: https://github.com/xxx/xxx.git
       dest: /home/user/
```
The existing code works but this a better way for cloning the repos using the git module and also wondering how these env values are being passed?